views/partials/_banner.njk

```diff
@@ -1,15 +1,24 @@
 {% from "phase-banner/macro.njk" import govukPhaseBanner %}
 
 {% if PULL_REQUEST %}
-  {% set pr_url = "https://github.com/alphagov/govuk-design-system/pull/" + REVIEW_ID %}
+  {% set prUrl = "https://github.com/alphagov/govuk-design-system/pull/" + REVIEW_ID %}
```

Comment:
Suggest this is either moved inside the `{% if REVIEW_ID %}` block or just omitted entirely and constructed within the `<a>` tag itself.

Reply:
would it be worth adding the branch name if no REVIEW_ID is present?

REPOSITORY_URL: URL to the Git repository the build pulls changes from.
BRANCH: Reference to check out after fetching changes from the Git repository.

Reply:
Sounds interesting, I'll give it a go.

Reply:
this is your current branch url https://**fix-master-netlify**--govuk-design-system-preview.netlify.com/